tests/test_diagnostics_database.py

```diff
@@ -48,9 +48,9 @@ def test_example_cdd(self):
 
         data = did.get_data_by_name('Diagnostic_Identification')
         self.assertEqual(data.name, 'Diagnostic_Identification')
-        self.assertEqual(data.start, 64)
+        self.assertEqual(data.start, 71)
         self.assertEqual(data.length, 16)
-        self.assertEqual(data.byte_order, 'little_endian')
+        self.assertEqual(data.byte_order, 'big_endian')
```

Comment:
Note to other reviewers: Diagnostic_Identification has datatype HexDump_2Byte. This has bo='21' which means big_endian.